pkg/corerp/frontend/controller/environments/createorupdateenvironment.go

```diff
@@ -152,10 +150,12 @@ func (e *CreateOrUpdateEnvironment) createOrUpdateACIEnvironment(ctx context.Con
 	resourceGroupName := rgID.FindScope("resourceGroups")
 	subscriptionID := rgID.FindScope("subscriptions")
 
-	// Ensure resource group is created.
-	err = clientv2.EnsureResourceGroupIsCreated(ctx, subscriptionID, resourceGroupName, ResourceLocation, &e.Options().Arm.ClientOptions)
+	// Get the location of the resource group
+	resourceLocation, err := clientv2.GetResourceGroupLocation(ctx, subscriptionID, resourceGroupName, &e.Options().Arm.ClientOptions)
 	if err != nil {
-		return err
+		// If resource group doesn't exist, use a default location
+		// For now, we'll return an error requiring the user to create the resource group first
+		return fmt.Errorf("failed to get resource group location. Please ensure the resource group exists: %w", err)
```

Comment:
The error message could be more actionable. Consider suggesting how users can create the resource group.